build_container/docker_build_linux.sh

```diff
@@ -18,7 +18,7 @@ config_env() {
 
 if [[ -z "${BUILD_TOOLS_PLATFORMS}" ]]; then
   if [[ "${OS_DISTRO}" == "ubuntu" ]]; then
-    export BUILD_TOOLS_PLATFORMS=linux/arm64,linux/amd64
+    export BUILD_TOOLS_PLATFORMS=linux/arm64,linux/amd64,linux/ppc64le
```

Comment:
let's hold off this for now as it drastically increases image build time in CI with emulation + building tools.

Reply:
Sure - I can remove this with the next commit I'm just wondering if you don't want to have it built at all or just while this PR is not finally approved.

Thanks!